app.js

```diff
@@ -1,54 +1,76 @@
 var hours = [
-  '10am: ',
-  '11am: ',
-  '12pm: ',
-  '1pm: ',
-  '2pm: ',
-  '3pm: ',
-  '4pm: ',
-  '5pm: '
+  '10am',
+  '11am',
+  '12pm',
+  '1pm',
+  '2pm',
+  '3pm',
+  '4pm',
+  '5pm'
 ];
+var stores = [];
+var tableView;
 
-Store.prototype.totals = [];
-Store.prototype.grandTotal = 0;
+// Store.prototype.totals = [];
+// Store.prototype.grandTotal = 0;
 
 function Store (storeName, minCust, maxCust, avgCookie, id) {
   this.storeName = storeName;
   this.minCust = minCust;
   this.maxCust = maxCust;
   this.avgCookie = avgCookie;
-  this.storeId = id;
+  this.totals = [];
+  this.grandTotal = 0;
+  stores.push(this); //pushes new stores (instances, when we use NEW STORE) into the stores array
+  Store.renderStore(tableView, this);
 };
 
 Store.prototype.getRandom = function(min, max) {
   return Math.random() * (max - min) + min;
 };
 
-Store.prototype.hourTotal = function(arr, total, avg, rand, hour) {
+Store.hourTotal = function(obj) {
   for(var j = 0; j < hours.length; j++) {
-    var monkey = Math.floor(this.getRandom(this.minCust, this.maxCust) * this.avgCookie);
-    this.totals.push(monkey);
-    total += monkey;
+    var monkey = Math.floor(obj.getRandom(obj.minCust, obj.maxCust)) * obj.avgCookie;
+    obj.totals.push(monkey);
+    obj.grandTotal += monkey;
   }
-  return total;
 };
 
-Store.prototype.render = function() {
-  Store.prototype.grandTotal = this.hourTotal(this.totals, this.grandTotal, this.avgCookie, this.getRandom, hours);
-  var pEl = document.getElementById(this.storeId);
-  var ulEl = document.createElement('ul');
-  pEl.textContent = this.storeName;
-
-  for (var i = 0; i < hours.length; i++) {
-    var liEl = document.createElement('li');
-    liEl.textContent = hours[i] + this.totals[i];
-    ulEl.appendChild(liEl);
+(Store.render = function() {
+  tableView = document.createElement('table');
+  tableView.setAttribute('id', 'theGrid');
```

Comment:
We talked about this in class yesterday, and I know you saw the difference, but you can also do the same thing by `tableView.id = 'theGrid';`